<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsChannelSettings</class>
 <widget class="QDialog" name="clsChannelSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>203</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>66666</width>
    <height>207</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(150,150,251);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="clsLabel" name="labelClose">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="images/res.qrc">:/close.ico</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelChannel">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>通道1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>238</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="labelItem1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>项目1：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelItem2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>项目2：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLimit1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>项目1上下限：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLimit2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>项目2上下限：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFreq">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>频率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLevel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>电平：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRange">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>档位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSpeed">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>速度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEqucct">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>等效电路：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelBias">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>偏置：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="labelItem11">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>L</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelItem22">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>Q</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLimit11">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>100uH--108uH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLimit22">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>6Q--88Q</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFreq1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>100kHz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLevel1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>1.00V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRange1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSpeed1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>Fast</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEqucct1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>Series</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelBias1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>0.5V,开</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>clsLabel</class>
   <extends>QLabel</extends>
   <header location="global">clsLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/res.qrc"/>
 </resources>
 <connections/>
</ui>
